app/views/admin/basic_projects/_form.html.slim

```diff
@@ -0,0 +1,46 @@
+= simple_form_for([:admin, @project],
+    html: { \
+      class: 'form-horizontal organization-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
+
+  = error_notification(f)
+
+  = f.input :name
+    .view-element = @project.try(:name)
+    = f.input_field :name
+
+  = f.input :status
+    .view-element = @project.try(:status)
+    = f.input_field :status_value, collection: BasicProject.status_options, include_blank: true
+
+  = authorized_form_field(simple_form: f, model: @project, field_name: :primary_agent,
+    choices: @agent_choices, classes: 'hidden-print')
+  = f.input t('activerecord.attributes.basic_project.primary_agent'), wrapper_html: {class: 'visible-print-block'}
```

Comment:
I'm not sure what's happening here with the field name being translated in the template. If you just put `:primary_agent` as the name, won't the translation happen automatically?